content/en/blog/falcosidekick-kubeless.md

```diff
@@ -0,0 +1,329 @@
+---
+title: Falcosidekick + Kubeless = a Kubernetes Response Engine
+date: 2021-01-15
+author: Thomas Labarussias
+---
+
+Two years ago, we presented to you a `Kubernetes Response Engine` based on `Falco`. The idea was to trigger [`Kubeless`](https://kubeless.io) serverless functions for deleting infected pod, start a `Sysdig` capture or forward the `events` to `GCP PubSub`. See the [README](https://github.com/falcosecurity/kubernetes-response-engine).
+
+To avoid maintaining this custom stack, we worked hard with the community for integrating all components into [`Falcosidekick`](https://github.com/falcosecurity/falcosidekick) and improving the UX. 
```

Comment:
```suggestion
To avoid maintaining this custom stack, we worked hard with the community to integrate all components into [`Falcosidekick`](https://github.com/falcosecurity/falcosidekick) and to improve the UX. 
```